plugins/terraform/terraform.go

```diff
@@ -19,7 +19,7 @@ func TerraformCLI() schema.Executable {
 			{
 				Description: "Credentials to use within the Terraform project",
 				SelectFrom: &schema.CredentialSelection{
-					ID:                    "project",
+					ID:                    "Project Credentials",
```

Comment:
What's the reason for this change? The previous version looks more like an ID to me.

Reply:
On `op` this ID is now used everywhere where the `Name` was used in the plugin commands, for messages like `Authenticate with __`. 
<img width="1428" alt="Screenshot 2023-05-19 at 13 21 13" src="https://github.com/1Password/shell-plugins/assets/45081667/4e1445d0-c33b-4996-b771-a0da58724f4f">

Reply:
e.g. `Authenticate with Terraform Project Credentials` sounds better.

Reply:
Yep, makes sense. Changed back to initial version.